helpers/controllers/userInfoController.ts

```diff
@@ -5,7 +5,7 @@ type Username = {
   username: string
 }
 
-const { User, UserLesson, Submission, Star } = db
+const { Lesson, User, UserLesson, Submission, Star } = db
```

Comment:
Can you separate out the back end and front end code for this PR? That will make things more organized and easier to review.